src/views/Checkout.vue

```diff
@@ -330,7 +330,11 @@ export default class Checkout extends Vue {
         transition: opacity 300ms, z-index 300ms;
     }
 
-    .account-info.active {
+    .merchant-info-shown > :not(.account-info) {
+        filter: blur(.75rem);
```

Comment:
@nibhar I think you discussed with Julian the exact blur amounts for the various elements. Can you help out here?